backend/wmg/pipeline/primary_filter_dimensions.py

```diff
@@ -109,14 +109,13 @@ def list_grouped_primary_filter_dimensions_term_ids(
 
 def order_tissues(ontology_term_ids: list[str]) -> list[str]:
     """
-    Order tissues based on appearance in TissueMapper.HIGH_LEVEL_TISSUES. This will maintain the priority set in
+    Order tissues based on appearance in HIGH_LEVEL_TISSUES. This will maintain the priority set in
     that class which is intended to keep most relevant tissues on top and tissues that are related to be placed
     sequentially
     """
     ontology_term_ids = set(ontology_term_ids)
```

Comment:
nit - this is mutating the original input - it might be better practice to do something like:

`unique_ontology_term_ids = set(ontology_term_ids)` and work off of that, makes it slightly more readable and easier to debug if needs be